src/cli/bootstrap/bootstrap.ts

```diff
@@ -79,10 +81,11 @@ export default async function bootstrap(args: string[]) {
         restoreCreateAccountFromMock();
 
         console.log('Change the key of the accounts to the new key', newPublicKey.toString());
-        await updateAccountKey('id.tonomy', newPublicKey, true);
+        await updateAccountKey('id.tmy', newPublicKey, true);
         await updateAccountKey('eosio.token', newPublicKey, true);
         await updateAccountKey('eosio', newPublicKey);
-
+        await updateAccountKey('found.tmy', newPublicKey, true);
+        await updateAccountKey('gov.tmy', newPublicKey, true, 'found.tmy');
```

Comment:
when i am trying to do something like this i am getting error.

[
  {
    message: `action's authorizations include a non-existent permission: {"actor":"gov.tmy","permission":"found.tmy"}`,
    file: 'transaction_context.cpp',
    line_number: 767,
    method: 'validate_referenced_accounts'
  }
]